package.json

```diff
@@ -82,7 +82,7 @@
   "dependencies": {
     "@google-cloud/pubsub": "^2.7.0",
     "@types/archiver": "^5.1.0",
-    "@types/filesize": "^5.0.0",
+    "JSONStream": "^1.2.1",
```

Comment:
ugh, different versions of npm order this differently, that's not annoying at all